src/template/intro.md

```diff
@@ -0,0 +1,70 @@
+# IN-CORE Notebook Gallery
+
+::::{grid} 2
+
+:::{grid-item}
+:columns: 4
+:class: sd-m-auto
+![ScreenShot](images/incore-logo.png)
+:::
+
+:::{grid-item}
+:columns: 8
+:class: sd-fs-4
+Submit your IN-CORE Jupyter Notebooks for peer-review to showcase your work. 
+:::
+
+:::{grid-item}
+:columns: 6
+:class: sd-fs-9
+Welcome to the IN-CORE Notebook Gallery. This gallery showcases jupyter notebooks that use the Interdependent Networked Community Resilience Modeling Environment ([IN-CORE](https://incore.ncsa.illinois.edu/)). All published notebooks in this gallery have been peer-reviewed. Notebooks are published on a quarterly-basis.
+:::
+
+:::{grid-item}
+:columns: 6
+:class: sd-fs-9
+To submit a notebook for publication in this gallery, please review the [submission guidelines](submission_guidelines.md) and [submission process](submission_process.md). 
+:::
+
+::::
+
+# Recently published notebooks
+
+Below are some recently published notebooks in the IN-CORE Gallery.
+
+
+::::{card-carousel} 3
+
+:::{card}
+:margin: 3
+:class-body: text-center
+:class-header: bg-light text-center
+:link: volumes/volume1/seaside-notebook/Seaside_testbed.html
```

Comment:
This should be dynamic eventually

Reply:
Definitely. I wasn't sure how to modify .md with python. Unless I hear otherwise, I can create an issue.

Reply:
Created issue #11 to automatically update `intro.md`. Commented out hardcoded part.

Reply:
Created issue #11 to automatically update `intro.md`. Commented out hardcoded part.